cmd/descheduler/app/server.go

```diff
@@ -94,6 +96,15 @@ func NewDeschedulerCommand(out io.Writer) *cobra.Command {
 				return
 			}
 
+			tracerProvider, err := tracing.NewTracerProvider(s.Tracing.CollectorEndpoint, s.Tracing.TransportCert, s.Tracing.ServiceName, s.Tracing.ServiceNamespace, s.Tracing.SampleRate)
+			if err != nil {
+				klog.Fatalf("failed to setup tracing provider", "err", err)
```

Comment:
should we really fail if we can't set up the tracing provider?

Reply:
Good question. I am not entirely sure what would be the right thing to do. One thing we can do is, switch back to a NoOp provider if we run into error and log that switch back here.